gitgud/operations.py

```diff
@@ -221,12 +221,11 @@ def get_operator():
     return None
 
 def init_tracking_json(file_operator):
-    if os.path.exists(os.path.join(file_operator.git_path, "gud", "commits.json")):
-        with open(os.path.join(file_operator.git_path, "gud", "commits.json"), 'w') as fp:
-            json.dump({}, fp)
-    else:
-        print("ERROR: Wrong directory :C")
-   
+    print("Creating: Commit Tracker")
+    with open(os.path.join(file_operator.git_path, "gud", "commits.json"), 'w') as fp:
```

Comment:
You can change this to file_operator.git_gud_path, but I would add an attribute to file_operator so you can do file_operator.commit_data_path

Reply:
Fixed in #130